webview-ui/package.json

```diff
@@ -28,18 +28,22 @@
 		"@radix-ui/react-slot": "^1.1.2",
 		"@radix-ui/react-tooltip": "^1.1.8",
 		"@tailwindcss/vite": "^4.0.0",
+		"@testing-library/dom": "^10.4.0",
```

Comment:
Curious why this is in `dependencies` rather than `devDependencies`.

Reply:
Wasn't actually needed at all 😬